### ISSUE TITLE
Status of testing Providers that were prepared on September 14, 2023

### ISSUE THREAD

**Issue by eladkal** [apache/airflow#34365]
### Body

I have a kind request for all the contributors to the latest provider packages release.
Could you please help us to test the RC versions of the providers?

The guidelines on how to test providers can be found in

[Verify providers by contributors](https://github.com/apache/airflow/blob/main/dev/README_RELEASE_PROVIDER_PACKAGES.md#verify-the-release-candidate-by-contributors)

Let us know in the comment, whether the issue is addressed.

Those are providers that require testing as there were some substantial changes introduced:

## Provider [amazon: 8.7.1rc1](https://pypi.org/project/apache-airflow-providers-amazon/8.7.1rc1)
   - [x] [Bugfix: Fix RDS triggers parameters so that they handle serialization/deserialization (#34222)](https://github.com/apache/airflow/pull/34222): @vandonr-amz
   - [x] [Use a AwsBaseWaiterTrigger-based trigger in EmrAddStepsOperator deferred mode (#34216)](https://github.com/apache/airflow/pull/34216): @yermalov-here
   - [x] [Remove unused parameter `cluster_role_arn` from `EksPodOperator`'s docstring (#34300)](https://github.com/apache/airflow/pull/34300): @Taragolis
   - [x] [Correct parameter names in docstring for S3CreateObjectOperator (#34263)](https://github.com/apache/airflow/pull/34263): @mjsqu
   - [x] [Simplify 'True if ... else False'  to bool(...) (#34258)](https://github.com/apache/airflow/pull/34258): @pankajkoti
## Provider [cncf.kubernetes: 7.6.0rc1](https://pypi.org/project/apache-airflow-providers-cncf-kubernetes/7.6.0rc1)
   - [x] [Add `progress_callback` parameter to `KubernetesPodOperator` (#34153)](https://github.com/apache/airflow/pull/34153): @zeotuan
   - [x] [Move definition of Pod*Exceptions to pod_generator (#34346)](https://github.com/apache/airflow/pull/34346): @potiuk
   - [x] [Push to xcom before `KubernetesPodOperator` deferral (#34209)](https://github.com/apache/airflow/pull/34209): @vchiapaikeo
## Provider [google: 10.9.0rc1](https://pypi.org/project/apache-airflow-providers-google/10.9.0rc1)
   - [x] [Stream (realtime) pipelines for CloudDataFusionStartPipelineOperator (#34271)](https://github.com/apache/airflow/pull/34271): @moiseenkov
   - [ ] [Add `expected_terminal_state` parameter to Dataflow operators (#34217)](https://github.com/apache/airflow/pull/34217): @shahar1
   - [ ] [Fix `ComputeEngineInsertInstanceOperator` doesn't respect jinja-templated instance name when given in body argument (#34171)](https://github.com/apache/airflow/pull/34171): @aryehgigi
   - [x] [fix: BigQuery job error message (#34208)](https://github.com/apache/airflow/pull/34208): @nathadfield
   - [ ] [fix(providers-google): gcp_conn_id parameter ignored in GKEStartPodOperator hook property (#34194)](https://github.com/apache/airflow/pull/34194): @mikaeld
   - [x] [Bump min common-sql provider version for Google provider (#34257)](https://github.com/apache/airflow/pull/34257): @pankajkoti
   - [x] [Remove unnecessary call to keys() method on dictionaries (#34260)](https://github.com/apache/airflow/pull/34260): @pankajkoti
   - [x] [Deprecate AutoMLTrainModelOperator for NL  (#34212)](https://github.com/apache/airflow/pull/34212): @VladaZakharova
   - [x] [Simplify 'True if ... else False'  to bool(...) (#34258)](https://github.com/apache/airflow/pull/34258): @pankajkoti
   - [x] [Google Dataform operators: Update `template_fields` (#34187)](https://github.com/apache/airflow/pull/34187): @retsilla

All users involved in the PRs:
@potiuk @shahar1 @vandonr-amz @pankajkoti @Taragolis @retsilla @zeotuan @aryehgigi @VladaZakharova @mikaeld @vchiapaikeo @nathadfield @mjsqu @moiseenkov @yermalov-here

### Committer

- [X] I acknowledge that I am a maintainer/committer of the Apache Airflow project.

**Comment by mjsqu** [comment]
#34263 - just a change to docstrings - no functional impact

**Comment by yermalov-here** [comment]
#34216 works as expected

**Comment by vchiapaikeo** [comment]
https://github.com/apache/airflow/pull/34209 continues to work as expected. Thank you @eladkal!

<img width="984" alt="image" src="https://github.com/apache/airflow/assets/9200263/ef58c3be-2b48-4ea8-b564-da2a8f227dac">

<img width="894" alt="image" src="https://github.com/apache/airflow/assets/9200263/a8d3e10d-1aa0-46ce-b55e-0a01c2d41dae">

**Comment by retsilla** [comment]
#34187 - works as expected.

**Comment by potiuk** [comment]
All good for the cncf.k8s fix.

**Comment by potiuk** [comment]
> While testing the sources for the Google RC, I see the dependences `apache-airflow-providers-common-sql>=1.7.2.dev0`,` apache-airflow>=2.4.0.dev0` have a `dev0` suffix in the `setup.cfg` file. Is that expected for RCs? I see `apache-airflow` has a `dev0` version suffix in the Amazon RC sources too in its `setup.cfg` file.
> Whereas for earlier released packages e.g. for the Google provider `10.8.0` it does NOT have a `dev0` suffix on the Airflow version in its setup.cfg file.

Yes. This is all expected (providing that your observations are about the RC packages you downloaded from PyPI).

I guess you downloaded the packages from PyPI - but those are not the packages that PMC members verify and vote on. They are "Convenience" to test and install the packages from PyPI but they are NOT the same as packges we vote on.

The RC packages in PyPI are different than the one we have in https://dist.apache.org/repos/dist/dev/airflow/providers/ - they differ by the RC* suffix and .dev0 dependencies. The packages in https://dist.apache.org/repos/dist/dev/airflow/providers/ are the very same files we will eventually upload as "x.y.z" packages to PyPI:

There are several different reasons for that process/differences:

1) In PyPI you cannot delete and re-upload package with the same version - so for PyPI testing we need to submit RC packages. It is impossible to have same package uploaded as RC and final version. So we need to slightly (dynamically) modify the packages when we are preparing them for PyPI (but only for RC candidates).

2) PIP does not recognize x.y.z.dev0 as `>=x.y.z`. This means that if we have two packages released at the same time and one depends on the other (happens) we need to dynamically add 'x.y.z.dev0` -> this will make sure that we can install the new packages from PyPI (otherwise they would comply that there is no `>=x.y.z` package. That''s why for all RCs we add `.dev0` dynamically for every dependency where we depend on any `apache-airflow` package. Otherwise we would have to first relase (vote and publish) the dependent package and only after the the package that depends on it. Ading .dev0 in PyPI RCS allows to vote and release such cross-dependent packages together.

3) Finally the packages we are voting on (because we do not vote on PyPI packages - we vote on the packages in https://dist.apache.org/repos/dist/dev/airflow/providers/ ) do not contain neither RC nor .dev0, because those are the very packages that we will move to https://downloads.apache.org/airflow/providers/ . They are THE SAME packages - with checksums and signatures etc. So the packages we VOTE on already have final versions, and final dependencies, because those are the packages that will become `x.y.z` if they pass the voting. 

So yeah - good points, but everything is perfectly as expected.

**Comment by vandonr-amz** [comment]
I'm go.

**Comment by zeotuan** [comment]
https://github.com/apache/airflow/pull/34153 - Works as expected

**Comment by nathadfield** [comment]
Hello,

Confirmed that #34208<https://github.com/apache/airflow/pull/34208> works as expected.

Cheers,

Nathan

From: Elad Kalif ***@***.***>
Date: Thursday, 14 September 2023 at 09:57
To: apache/airflow ***@***.***>
Cc: Nathan Hadfield ***@***.***>, Mention ***@***.***>
Subject: [apache/airflow] Status of testing Providers that were prepared on September 14, 2023 (Issue #34365)
Body I have a kind request for all the contributors to the latest provider packages release. Could you please help us to test the RC versions of the providers? The guidelines on how to test providers can be found in Verify providers by contributors
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.
ZjQcmQRYFpfptBannerEnd
Body

I have a kind request for all the contributors to the latest provider packages release.
Could you please help us to test the RC versions of the providers?

The guidelines on how to test providers can be found in

Verify providers by contributors<https://urldefense.com/v3/__https:/github.com/apache/airflow/blob/main/dev/README_RELEASE_PROVIDER_PACKAGES.md*verify-the-release-candidate-by-contributors__;Iw!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqD7sIRmo$>

Let us know in the comment, whether the issue is addressed.

Those are providers that require testing as there were some substantial changes introduced:

Provider amazon: 8.7.1rc1<https://urldefense.com/v3/__https:/pypi.org/project/apache-airflow-providers-amazon/8.7.1rc1__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xq8lBieWI$>

  *   Bugfix: Fix RDS triggers parameters so that they handle serialization/deserialization (#34222)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34222__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xq797PC6g$>: @vandonr-amz<https://urldefense.com/v3/__https:/github.com/vandonr-amz__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xqtbgb6Q4$>
  *   Use a AwsBaseWaiterTrigger-based trigger in EmrAddStepsOperator deferred mode (#34216)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34216__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xq1UOYZw0$>: @yermalov-here<https://urldefense.com/v3/__https:/github.com/yermalov-here__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqmL2v8VU$>
  *   Remove unused parameter cluster_role_arn from EksPodOperator's docstring (#34300)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34300__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqS5mGQfg$>: @Taragolis<https://urldefense.com/v3/__https:/github.com/Taragolis__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqVGEnQJ8$>
  *   Correct parameter names in docstring for S3CreateObjectOperator (#34263)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34263__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqZ4ORuXY$>: @mjsqu<https://urldefense.com/v3/__https:/github.com/mjsqu__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqNS9K6W8$>
  *   Simplify 'True if ... else False' to bool(...) (#34258)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34258__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqXkiBY-c$>: @pankajkoti<https://urldefense.com/v3/__https:/github.com/pankajkoti__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqzbN90bE$>

Provider cncf.kubernetes: 7.6.0rc1<https://urldefense.com/v3/__https:/pypi.org/project/apache-airflow-providers-cncf-kubernetes/7.6.0rc1__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqPZ0TaEI$>

  *   Add progress_callback parameter to KubernetesPodOperator (#34153)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34153__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqbD7IJOI$>: @zeotuan<https://urldefense.com/v3/__https:/github.com/zeotuan__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqFldyc4s$>
  *   Move definition of Pod*Exceptions to pod_generator (#34346)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34346__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqO1D3hc0$>: @potiuk<https://urldefense.com/v3/__https:/github.com/potiuk__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqcqiCI_E$>
  *   Push to xcom before KubernetesPodOperator deferral (#34209)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34209__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqsFc1R5w$>: @vchiapaikeo<https://urldefense.com/v3/__https:/github.com/vchiapaikeo__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqaEIumSs$>

Provider google: 10.9.0rc1<https://urldefense.com/v3/__https:/pypi.org/project/apache-airflow-providers-google/10.9.0rc1__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqonQQ7ws$>

  *   Stream (realtime) pipelines for CloudDataFusionStartPipelineOperator (#34271)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34271__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqMmgcHMM$>: @moiseenkov<https://urldefense.com/v3/__https:/github.com/moiseenkov__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqWz1xwT8$>
  *   Add expected_terminal_state parameter to Dataflow operators (#34217)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34217__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqhNEvEN0$>: @shahar1<https://urldefense.com/v3/__https:/github.com/shahar1__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqNGdte28$>
  *   Fix ComputeEngineInsertInstanceOperator doesn't respect jinja-templated instance name when given in body argument (#34171)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34171__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xqfq264oA$>: @aryehgigi<https://urldefense.com/v3/__https:/github.com/aryehgigi__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xq3M6zzYA$>
  *   fix: BigQuery job error message (#34208)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34208__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqKm9j0Bo$>: @nathadfield<https://urldefense.com/v3/__https:/github.com/nathadfield__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xq0eHUCWQ$>
  *   fix(providers-google): gcp_conn_id parameter ignored in GKEStartPodOperator hook property (#34194)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34194__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xq6A5lgYs$>: @mikaeld<https://urldefense.com/v3/__https:/github.com/mikaeld__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqmJPgpvQ$>
  *   Bump min common-sql provider version for Google provider (#34257)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34257__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqsJchFjQ$>: @pankajkoti<https://urldefense.com/v3/__https:/github.com/pankajkoti__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqzbN90bE$>
  *   Remove unnecessary call to keys() method on dictionaries (#34260)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34260__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqDViyXfY$>: @pankajkoti<https://urldefense.com/v3/__https:/github.com/pankajkoti__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqzbN90bE$>
  *   Deprecate AutoMLTrainModelOperator for NL (#34212)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34212__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqTavIvQg$>: @VladaZakharova<https://urldefense.com/v3/__https:/github.com/VladaZakharova__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqqwfXu7Y$>
  *   Simplify 'True if ... else False' to bool(...) (#34258)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34258__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqXkiBY-c$>: @pankajkoti<https://urldefense.com/v3/__https:/github.com/pankajkoti__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqzbN90bE$>
  *   Google Dataform operators: Update template_fields (#34187)<https://urldefense.com/v3/__https:/github.com/apache/airflow/pull/34187__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqmKxkDSI$>: @retsilla<https://urldefense.com/v3/__https:/github.com/retsilla__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xq3n8OUZA$>

All users involved in the PRs:
@potiuk<https://urldefense.com/v3/__https:/github.com/potiuk__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqcqiCI_E$> @shahar1<https://urldefense.com/v3/__https:/github.com/shahar1__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqNGdte28$> @vandonr-amz<https://urldefense.com/v3/__https:/github.com/vandonr-amz__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xqtbgb6Q4$> @pankajkoti<https://urldefense.com/v3/__https:/github.com/pankajkoti__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqzbN90bE$> @Taragolis<https://urldefense.com/v3/__https:/github.com/Taragolis__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqVGEnQJ8$> @retsilla<https://urldefense.com/v3/__https:/github.com/retsilla__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xq3n8OUZA$> @zeotuan<https://urldefense.com/v3/__https:/github.com/zeotuan__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqFldyc4s$> @aryehgigi<https://urldefense.com/v3/__https:/github.com/aryehgigi__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xq3M6zzYA$> @VladaZakharova<https://urldefense.com/v3/__https:/github.com/VladaZakharova__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqqwfXu7Y$> @mikaeld<https://urldefense.com/v3/__https:/github.com/mikaeld__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqmJPgpvQ$> @vchiapaikeo<https://urldefense.com/v3/__https:/github.com/vchiapaikeo__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqaEIumSs$> @nathadfield<https://urldefense.com/v3/__https:/github.com/nathadfield__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7Xq0eHUCWQ$> @mjsqu<https://urldefense.com/v3/__https:/github.com/mjsqu__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqNS9K6W8$> @moiseenkov<https://urldefense.com/v3/__https:/github.com/moiseenkov__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqWz1xwT8$> @yermalov-here<https://urldefense.com/v3/__https:/github.com/yermalov-here__;!!Ci6f514n9QsL8ck!mya6uz52Q94M3u-PyuiPCGjfDX4t7C_5H_cCMZA5JS6V3ouZZ-Fpgqo5Zd3grv0AHmkCbT4G1VuvBH5mV7XqmL2v8VU$>

Committer

  *   I acknowledge that I am a maintainer/committer of the Apache Airflow project.

**Comment by eladkal** [comment]
Thank you everyone.
Providers are released
I invite everyone to help improve providers for the next release, a list of open issues can be found [here](https://github.com/apache/airflow/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%3Aproviders).
